src/runtime.rs

```diff
@@ -139,6 +159,16 @@ where
         db.for_each_query(|query_storage| query_storage.sweep(db, strategy));
     }
 
+    /// True if `d` represents the max durability level.
+    pub(crate) fn is_constant(&self, d: Durability) -> bool {
+        d == self.max_durability()
+    }
```

Comment:
unused function. 

Should we remove `allow(unused)` from lib.rs?

Reply:
I presume you mean `#[allow(dead_code)]` -- but yeah, we should.